LaTeX/TeX.sublime-syntax

```diff
@@ -111,6 +118,46 @@ contexts:
       captures:
         1: punctuation.definition.backslash.tex
 
+  registers:
+    - match: (\\)({{registers}}){{endcs}}
+      captures:
+        1: punctuation.definition.backslash.tex
+        2: storage.type.tex
+      push:
+        - meta_scope: variable.language.tex
+        - include: register-id
+
+  register-id:
+    - match: \d+
+      scope: constant.numeric.tex
```

Comment:
I'd suggest to add `.value` to be specific enough in case a color scheme addresses all parts of numeric constants (`constant.numeric.[base|value|suffix]`).

```suggestion
      scope: constant.numeric.value.tex
```

Reply:
Maybe even add `meta.number.integer.decimal` to fully comply with the other syntaxes.